collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java

```diff
@@ -428,28 +428,28 @@ public void onClick(DialogInterface dialog, int which) {
 
     public class DrawView extends View {
         private boolean isSignature;
-        private Bitmap mBitmap;
-        private Canvas mCanvas;
-        private Path mCurrentPath;
-        private Path mOffscreenPath; // Adjusted for position of the bitmap in the view
-        private Paint mBitmapPaint;
-        private File mBackgroundBitmapFile;
-        private float mX;
-        private float mY;
+        private Bitmap bitmap;
+        private Canvas canvas;
+        private Path currentPath;
+        private Path offscreenPath; // Adjusted for position of the bitmap in the view
+        private Paint bitmapPaint;
+        private File backgroundBitmapFile;
+        private float x1;
```

Comment:
xCoordinate can't be used as it does not follow the naming convention. So using latestX and latestY would be better